include/k4a/k4a.hpp

```diff
@@ -845,6 +845,31 @@ class transformation
         }
     }
 
+    /** Transforms depth map and an custom image into the geometry of the color camera.
+     * Throws error on failure
+     *
+     * \sa k4a_transformation_custom_depth_image_to_color_camera
+     */
+    void custom_depth_image_to_color_camera(const image &depth_image,
```

Comment:
>custom_depth_image_to_color_camera [](start = 9, length = 34)

nit: may be change to depth_custom_image_to_color_camera

Reply:
I will change this to mimic whatever name is defined in k4a.h.